sdk/core/Azure.Core.TestFramework/src/TestServer.cs

```diff
@@ -45,8 +45,8 @@ public TestServer(RequestDelegate app, bool https = false)
                     services.AddSingleton<IStartup>(this);
                 })
                 .UseSetting(WebHostDefaults.ApplicationKey, typeof(TestServer).GetTypeInfo().Assembly.FullName)
+                .UseShutdownTimeout(TimeSpan.FromMilliseconds(1000))
```

Comment:
I may have fixed both tests that required this.

Reply:
Is there any down side to keeping it?

Reply:
I've discovered a few response lifetime bugs because kestrel took a long time to shut down signaling that not all connections were closed.